core/src/main/scala/org/apache/spark/storage/FallbackStorage.scala

```diff
@@ -155,6 +153,23 @@ private[spark] object FallbackStorage extends Logging {
       FALLBACK_BLOCK_MANAGER_ID, blockId, StorageLevel.DISK_ONLY, memSize = 0, dataLength)
   }
 
+  /**
+   * Provide the Path for a shuffle file.
+   */
+  private[storage] def getPath(conf: SparkConf,
+                               appId: String,
+                               shuffleId: Int,
+                               filename: String): Path = {
+    val fallbackPath = new Path(conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH).get)
+    val subPaths = conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_SUBPATHS)
```

Comment:
It would have been better to pull these `conf.get` as fields and move method into the class ... but looks like `FallbackStorage.read` needs it too unfortunately.